### PR TITLE
[ASAP-430]- team coproduction from cms

### PR DESCRIPTION
**Pull request by AkosuaA**
I combined the 'within team' and 'across team' tickets.
ref: https://asaphub.atlassian.net/browse/ASAP-430 & https://asaphub.atlassian.net/browse/ASAP-429

Flag to display Collaboration tab: `document.cookie = 'ASAP_DISPLAY_ANALYTICS_COLLABORATION=true';`

### CODE REVIEW

File: apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts [hunk 1]
```diff
@@ -83,3 +93,455 @@ describe('collaboration', () => {
...
+              Protocol: 0,
+            },
+            byTeam: [],
+          },
+        }),
+      ],
+    });
+  });
+  describe('within team', () => {
+    test('Should return a count of 2 if client has two outputs of the same document type with multiple labs', async () => {
```
Comment (peterstarling) [inline apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts:260]: it seems to have three outputs of type 'Article' though, no?

File: apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts [hunk 2]
```diff
@@ -83,3 +93,455 @@ describe('collaboration', () => {
...
+          }),
+        ],
+      });
+    });
+  });
+
+  describe('across teams', () => {});
+  test('Should return a count of 2 if client has two outputs of the same document type with multiple teams', async () => {
+    const graphqlResponse = getTeamCollaborationQuery();
+    const researchOutputs = [
```
Comment (peterstarling) [inline apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts:341]: maybe you want to add an output which has only a single team to show it does not count?

File: apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts [hunk 3]
```diff
@@ -83,3 +93,455 @@ describe('collaboration', () => {
...
+                Protocol: 0,
+              },
+            ],
+          },
+        }),
+      ],
+    });
+  });
+
+  test('Should group outputs by team', async () => {
```
Comment (peterstarling) [inline apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts:435]: grouped sure but also is there a sort order?

File: apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts (hunk 1 above)
Comment (AkosuaA) [reply apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts:260]: No, it's 2. One has its addedDate outside the 30 day range

File: apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts (hunk 2 above)
Comment (AkosuaA) [reply apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts:341]: Added

**Pull request merged**